.azure-pipelines/steps.yml

```diff
@@ -28,7 +28,7 @@ steps:
   displayName: 'gem install rspec_junit_formatter'
 
 - script: |
-    timeout 50m bash -c 'ruby -r rspec_junit_formatter bin/rspec --format progress --format RspecJunitFormatter -o rspec/bundler-junit-results.xml' || exit 0
```

Comment:
`timeout 120m bash -c ...` maybe?

Reply:
When I wrapped this with `timeout`, it was to intercept Azure's global timeout of 1h, and being able to always produce a green build. But maybe the global timeout is higher now? I'll give it a try.

Reply:
Oh, 1h limit? Meh.

Did it work? Can't see because of the force push :/

If not, the best solution is probably to just deal with it and fix those failures :D

Reply:
I didn't try, actually. All I did was increasing the allowed failures to 250 on windows and it took 40 minutes to reach to that number of failures, so it didn't time out.

Reply:
Found the proper configuration, now all Azure jobs will run until the end! :+1: